common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -99,8 +99,15 @@ public static UTF8String fromBytes(byte[] bytes, int offset, int numBytes) {
   /**
    * Creates an UTF8String from given address (base and offset) and length.
    */
-  public static UTF8String fromAddress(Object base, long offset, int numBytes) {
-    return new UTF8String(base, offset, numBytes);
+  public static UTF8String fromAddress(MemoryBlock base, long offset, int numBytes) {
```

Comment:
can we remove this method and ask the caller side to do
```
UTF8String.fromMemoryBlock(block.subBlock(offset, length))
```

Reply:
Good catch. We do not want to use address.